mod_reforged/config/indirect_numeral_adjectives.nut

```diff
@@ -0,0 +1,202 @@
+::Reforged.Numerals <- {
+	// Config
+	ExactPartySize = false,
+	ExactDialogSize = false,
+	PartyRange = false,
+	DialogRange = true
+
+	// Data
+	IndefiniteNumerals = [],
+
+	function register( _name, _startsAt, _suffix = "" )
+	{
+		::Reforged.Numerals.IndefiniteNumerals.push({
+			Name = _name,
+			StartsAt = _startsAt,
+			Suffix = _suffix,
+			Range = ""
+		});
+	}
+
+	function setExactPartySize( _bool )
+	{
+		this.ExactPartySize = _bool;
+		this.updateAllParties();
+	}
+
+	function setPartyRange( _bool )
+	{
+		this.PartyRange = _bool;
+		this.updateAllParties();
+	}
+
+
+	// This always needs to be called after all new numerals of your mod are registered.
+	// This will sort all Numerals and generate their ranges automatically
+	function calculateRanges()
```

Comment:
One can make it so that the numerals must be registered during mod queueing. Which then means that this `calculateRanges()` can be called by us automatically during msu `AfterQueue` (or the AfterQueue bucket in Modern Hooks once we switch to that). Then modders don't have to call it.